### PR TITLE
Upgrade 0x to 9.0-prerelease for WSS support

### PR DESCRIPTION
**Pull request by pgebheim**

### CODE REVIEW

**Comment by pgebheim** [comment]
Still need to regression test this locally (running other stuff off this branch now)

File: packages/augur-tools/src/flash/scripts.ts [hunk 1]
```diff
@@ -1885,13 +1886,33 @@ export function addScripts(flash: FlashSession) {
         description: 'Name of contract',
         required: true,
       },
+      {
+        name: 'removePrefix',
```
Comment (robert-e-davidson3) [inline packages/augur-tools/src/flash/scripts.ts]: Why not check the first two characters and remove `0x` if present? Less prone to config issues.

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). ❤️
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/m96hxno**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

**Review by robert-e-davidson3 — APPROVED** [review]
j'approuve

**Pull request merged**